src/Gridify/Builder/LinqQueryBuilder.cs

```diff
@@ -23,7 +23,8 @@ public class LinqQueryBuilder<T>(IGridifyMapper<T> mapper) : BaseQueryBuilder<Ex
                   targetExp.Parameters[0],
                   value,
                   op,
-                  gMap.Convertor);
+                  gMap.Convertor,
+                  true);
```

Comment:
also this parameter can be removed

Reply:
I needed this parameter otherwise List<string> matching was failing. 
The value parameter was becoming ToLower but List<string> left side was not.

Reply:
Yes I know the thing is this is related to refactoring, from the next version we don't use `ToLower` anymore. we use StringComparison instead to improve the quality and performance. it is ok if it fails now. I'll transfer my changes to your branch